openslides/agenda/urls.py

```diff
@@ -4,14 +4,7 @@
 
 urlpatterns = patterns(
     '',
-
-    # PDF
     url(r'^print/$',
         views.AgendaPDF.as_view(),
         name='agenda_pdf'),
-
-    # TODO: remove it after implement projector rest api
```

Comment:
This is a special case for me (because this was an implementation for a customer) so I need an extra issue here.